rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```diff
@@ -281,12 +300,14 @@ static bool encl_build(struct sgx_secs *secs, void *bin, unsigned long bin_size,
 
 		enclave_fd = dev_fd;
 	}
-
+	free(ssa_frame);
 	return true;
 out_map:
 	munmap((void *)secs->base, secs->size);
+	free(ssa_frame);
```

Comment:
free() should be called before munmap in the reverse order determined by caller.